.github/workflows/publish-release.yml

```diff
@@ -23,7 +33,25 @@ jobs:
           packageUser: ${{ github.actor }}
           packagePAT: ${{ secrets.BALLERINA_STDLIB_TOKEN }}
         run: |
-          NEW_VERSION=$(echo "${GITHUB_REF}" | cut -d "/" -f3)
-          echo "New version: ${NEW_VERSION}"
-          echo "Github username: ${GITHUB_ACTOR}"
-          ./gradlew -PpublishToCentral=true -Pversion=${NEW_VERSION} publish
+          ./gradlew release -Prelease.useAutomaticVersion=true
```

Comment:
@niveathika don't we need `./gradlew publish` here?

Reply:
During the release only we have to publish dont we? After the version of the module also removed SNAPSHOT?

Reply:
Does, `./gradle release` publish to GitHub packages and central(if need)?. IINM, it doesn't.

Reply:
Yes, publish is called right after the build is checked, https://github.com/ballerina-platform/module-ballerina-sql/blob/de783004cce57f5d57789a581079349c6fa0a76a/build.gradle#L164